collectfast/tests/command/test_disable.py

```diff
@@ -3,14 +3,13 @@
 
 from django.test import override_settings as override_django_settings
 
+from collectfast.tests.command.utils import call_collectstatic
```

Comment:
This doesn't seem like a justified change, let's keep this a relative import.

Reply:
I thought the absolute path was more straightforward. But in this case, I guess relative is enough. 😄